src/uu/mkdir/src/mkdir.rs

```diff
@@ -124,6 +156,15 @@ pub fn uu_app() -> Command {
                 .help("print a message for each printed directory")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::SELINUX)
+                .short('Z')
+                .help("set SELinux security context of each created directory to the default type")
+                .action(ArgAction::SetTrue),
+        )
+        .arg(Arg::new(options::CONTEXT).long(options::CONTEXT).help(
```

Comment:
To match the help text:
```suggestion
        .arg(Arg::new(options::CONTEXT).long(options::CONTEXT).value_name("CTX").help(
```